internal/redis/model.go

```diff
@@ -0,0 +1,12 @@
+package redis
+
+import (
+	"context"
+	"time"
+)
+
+type MemStorage interface {
+	Ping(ctx context.Context) error
```

Comment:
It should not be a part of `MemStorage`.

Reply:
fixed